lib/eslint/eslint.js

```diff
@@ -881,15 +881,6 @@ class ESLint {
 
                 const config = configs.getConfig(filePath);
 
-                /*
-                 * Sometimes a file found through a glob pattern will
-                 * be ignored. In this case, `config` will be undefined
-                 * and we just silently ignore the file.
-                 */
-                if (!config) {
-                    return void 0;
-                }
```

Comment:
This check is the same as `if (ignored)` on line 874. `ignored` is `true` whenever a file has no config, even if it's not ignored by a pattern.